paws_collector.js

```diff
@@ -96,14 +96,15 @@ class PawsCollector extends AlAwsCollector {
         }
     }
 
-    constructor(context, {aimsCreds, pawsCreds}, childVersion, healthChecks = [], statsChecks = []) {
+    constructor(context, {aimsCreds, pawsCreds}, childVersion,healthChecks = [], statsChecks = []) {
         const version = childVersion ? childVersion : packageJson.version;
         const endpointDomain = process.env.paws_endpoint.replace(DOMAIN_REGEXP, '');
+        let collectorStreams = process.env.collector_stream ? JSON.parse(process.env.collector_stream) : [];
```

Comment:
please add one more extra check to make sure `JSON.parse(process.env.collector_stream)` is an array